app/models/requests/ApplicationSummaryRequest.scala

```diff
@@ -34,7 +43,9 @@ object ApplicationSummaryResponse {
 
 final case class ApplicationSummary(
   id: ApplicationId,
-  holder: EORIDetails
+  nameOfGoods: String,
+  dateSubmitted: Instant,
+  eoriNumber: EORI
```

Comment:
I wonder if EORI number may be considered as redundant info considering it will be the same for all application summaries retrieved. I could be wrong though.

Reply:
Ah yes, James pointed out that in future an agent might have multiple applications across different traders (with different EORI numbers)

Reply:
Future-proofing. What if the future never happens  😄 ?